test/cmdlineTests/name_simplifier/output

```diff
@@ -119,14 +125,20 @@ object "C_56" {
                 let shiftBits := mul(vloc, 8)
                 let mask := shl(shiftBits, not(0))
                 sstore(slot, or(and(_6, not(mask)), and(shl(shiftBits, _5), mask)))
-                let _7, _8 := storage_array_index_access$_t_struct$_S_storage(0x02, vloc)
+                let _7, _8 := storage_array_index_access$_t_struct$_S_storage_1220()
                 vloc := extract_from_storage_value_dynamict_uint256(sload(_7), _8)
                 vloc__24_mpos := convert_t_stringliteral_6490_to_t_string()
             }
-            function storage_array_index_access$_t_struct$_S_storage(array, index) -> slot, offset
+            function storage_array_index_access$_t_struct$_S_storage_1219() -> slot, offset
+            {
+                if iszero(lt(slot, 0x02)) { invalid() }
```

Comment:
It's weird that this is not optimized.

Reply:
https://gist.github.com/hrkrshnn/71c49684c5ae60974f5bed90f40fb73a is the AST after it's running the "[]" part of full suite. Using yulopti, we don't get to the point where the condition is getting simplified to 0.

Reply:
Also has nothing to do with reaching the max for `maxRounds`. In fact 'runSequenceUntilStable' stops after just 4 rounds.